examples/experimental/openmmforcefields/gaff.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    imported = from_openmm(
</pre><p>Could be good to clarify that this is <code>imported_interchange</code> - the type is heavily implied but it couldn't hurt to be more explicit.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-interchange/pull/857/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>